src/components/Alert/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { AlertState } from '../../stores/appStore';
+import { Alert as MTAlert, Text } from '@mantine/core';
+import { useStore } from '../../useStore';
+import styles from './styles.module.css';
+
+function Alert({ message, type = 'info', autoClose = true }: AlertState) {
```

Comment:
Alert props does not need to be passed but they can be read from store directly?